.github/workflows/scripts/pentest_ingest_scans.py

```
@@ -0,0 +1,88 @@
+"""
+import json
+import os
+import subprocess
+import sys
+
+REPO = os.environ.get("GITHUB_REPOSITORY", "")
+TAG = os.environ.get("PENTEST_RELEASE_TAG", "nightly")
+OUT = os.environ.get("PENTEST_CONTEXT", "context.json")
+
+
+def gh(*args):
```

Ruff (0.16.1)

[warning] Missing type annotation for `*args`

(ANN002)